src/components/ebay-bar-chart/component.js

```diff
@@ -0,0 +1,343 @@
+import Highcharts from 'highcharts';
+import accessibility from 'highcharts/modules/accessibility';
+import patternFill from 'highcharts/modules/pattern-fill';
+
+import {
+    backgroundColor,
+    gridColor,
+    labelsColor,
+    legendColor,
+    legendInactiveColor,
+    legendHoverColor,
+    tooltipBackgroundColor,
+    tooltipShadows,
+    setSeriesColors,
+} from '../../common/charts/colors';
+
+import { ebayLegend } from '../../common/charts/legend';
+import { eBayColumns } from '../../common/charts/bar-chart';
+
+if (typeof Highcharts === 'object') {
```

Comment:
If highcharts is not loaded in the page or if they load it later, then this won't be executed.
Maybe this should be put in an `onMount` and use `DOMContentedLoaded` or `load` event (see video `onMount`)

Reply:
I have this conditional here so that it doesn't run on the server side.  I can see if I can move it into the onMount.  Not sure if it will cause any problems or not

Reply:
The issue isn't for server. It's on the client timing. This might not be called in time if `Highcharts` loads after this code. Some pages do optimizations where they load the code after page load to help site speed.

Reply:
So looking at the video player it is loading it's js manually from an ebaystatic url.  Are we planning to try and break highcharts out into a separate JS file to do this?  It tree shakes rather well when compiling the component.

Reply:
So the thing is, highcharts is large. This could cause issues on loading the page if it's too big. I know with video this was a major concern on pages. 
We could change it like video to load it after page load.